core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.ts

```diff
@@ -25,27 +27,36 @@ export class EditContentLayoutComponent {
 
     private readonly dotEditContentService = inject(DotEditContentService);
     isContentSaved = false;
-    formData$ = this.identifier
+
+    formData$: Observable<EditContentFormData> = this.identifier
         ? this.dotEditContentService.getContentById(this.identifier).pipe(
-              switchMap(({ contentType }) => {
+              switchMap(({ contentType, ...contentData }) => {
                   if (contentType) {
                       this.contentType = contentType;
 
-                      return this.dotEditContentService.getContentTypeFormData(contentType);
+                      return this.dotEditContentService
+                          .getContentTypeFormData(contentType)
+                          .pipe(map((res) => ({ values: { ...contentData }, layout: res })));
```

Comment:
```suggestion
                          .pipe(map((res) => ({ contentlet: { ...contentData }, layout: res })));
```

According to our talk today.